event-driven-with-argo/README.md

```diff
@@ -1,57 +1,89 @@
- 
-``` 
-helm repo add argo https://argoproj.github.io/argo-helm
-helm repo add localstack https://helm.localstack.cloud 
-```
+# Event driven with argo:
+## Installation
 
+- The user must start a Minikube cluster (you can use the `--driver` option to specify the container runtime) using the command below:  
 
-```python
+    ```
+    minikube start --driver=docker
+    ```  
 
-from os import environ
-from redis import Redis
-from time import sleep
+    Using `--driver=docker` runs Minikube inside a Docker container instead of a virtual machine, which provides several advantages. It ensures a faster and more lightweight Kubernetes cluster initialization. This approach is particularly useful for development and testing environments where quick deployment and easy cleanup.
 
-stream_key = environ.get("STREAM", "STREAM")
-producer = environ.get("PRODUCER", "user-1")
+- Please add the following repositories to your Helm chart to ensure access to the required packages:  
 
+    ```sh
+    helm repo add argo https://argoproj.github.io/argo-helm
+    helm repo add localstack https://helm.localstack.cloud
+    ```  
 
-def connect_to_redis():
-    hostname = environ.get("REDIS_HOSTNAME", "redis-service") 
-    port = environ.get("REDIS_PORT", 6379)
+    Adding these repositories allows you to seamlessly install and manage **Argo** (a workflow orchestration tool) and **LocalStack** (a local AWS cloud emulator) using Helm. This ensures that your Kubernetes environment is properly configured for deploying workflows and simulating AWS services efficiently.
+- Run the command below:
+    ```
+    skaffold dev
+    ```
 
-    return Redis(hostname, port, retry_on_timeout=True)
+- Wait for the deployment to stablize (1-3 minutes) and then open your browser on the link printed, usually http://127.0.0.1:8000. 
 
+- You will see the code server is running. Follow the documentations under `doc` folder.
 
-def send_event(redis_connection, aoi, reference):
-    count = 0
 
-    try:
-        # TODO cloud events
-        # un-map the "data" wrt app package parameters
-        data = {
-            "producer": producer,
-            "href": reference,
-        }
-        resp = redis_connection.xadd(stream_key, data)
-        print(resp)
-        count += 1
+> Note: Here is an example of Redis' publisher which is discussed in detail in the repository you have on code server
```

Comment:
This not the normal way to consume "eoap" documentation resources. Use this link https://eoap.github.io/event-driven-with-argo/ instead